helpers/micromark-helpers.cjs

```diff
@@ -284,6 +284,18 @@ const nonContentTokens = new Set([
   "undefinedReferenceShortcut"
 ]);
 
+/**
+ * Determines if a link is a valid link (and not a fake shortcut link due to parser tricks).
+ *
+ * @param {Token} label Label token.
+ * @param {Token} labelText LabelText token.
+ * @param {Map<string, any>} definitions Map of link definitions.
+ * @returns {boolean} True iff the link is valid.
+ */
+function validLink(label, labelText, definitions) {
```

Comment:
I think this function is no longer necessary after a recent refactoring. I'm about to remove it from the code, so will question whether it's necessary in your scenario, either.

Reply:
It's gone and you should be able to simplify similarly: https://github.com/DavidAnson/markdownlint/commit/637975f517cfe856d034cc27951cf2afaf4186a3